src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -31,10 +31,36 @@ namespace Microsoft.ML.Transforms.Text
     /// </summary>
     public interface IStopWordsRemoverOptions { }
 
-    // A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are counts
-    // of (word or character) ngrams in a given text. It offers ngram hashing (finding the ngram token string name to feature
-    // integer index mapping through hashing) as an option.
-    /// <include file='doc.xml' path='doc/members/member[@name="TextFeaturizingEstimator "]/*' />
+    /// <summary>
+    ///  A transform that turns a collection of text documents into numerical feature vectors.
+    ///  The feature vectors are normalized counts of word and/or character ngrams (based on the options supplied) in a given tokenized text.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes. |
+    /// | Input column data type | [text](xref:System.ReadOnlyMemory<char[]>) |
+    /// | Output column data type | vector of <xref:System.Single> |
+    ///
+    /// ]]>
+    ///
+    /// This estimator gives the user one-stop solution for doing:
+    /// * Language Detection
+    /// * [Tokenzation](https://en.wikipedia.org/wiki/Lexical_analysis#Tokenization)
+    /// * [Text normalization](https://en.wikipedia.org/wiki/Text_normalization)
+    /// * [Predefined and custom stopwords removal](https://en.wikipedia.org/wiki/Stop_words)
+    /// * [Word-based or character-based Ngram and SkipGram extraction](https://en.wikipedia.org/wiki/N-gram)
+    /// * [TF, IDF or TF-IDF](https://en.wikipedia.org/wiki/Tf%E2%80%93idf)
+    /// * [L-p vector normalization](xref: Microsoft.ML.Transforms.LpNormNormalizingTransformer)
+    ///
+    ///  Features are made of (word/character) n-grams/skip-grams​ and the number of features are equal to the vocabulary size found by analyzing the data.
+    ///     </format>
```

Comment:
I believe you can specify the maximum number of ngrams to keep in the advanced options.

Reply:
Maybe you could start by: `By default, ...`.
And afterwards say that: `the number of features can also be specified by selecting the maximum number of n-gram to keep in the advanced options`. 

---
In reply to: [277060289](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277060289) [](ancestors = 277060289)